mantle/cosa/cosa_v1.go

```diff
@@ -1,6 +1,6 @@
 package cosa
 
-// generated by "schematyper ../src/schema/v1.json -o cosa/cosa_v1.go --package=cosa --root-type=Build" -- DO NOT EDIT
+// generated by "/src/cosa/mantle/bin/schematyper ../src/schema/v1.json -o cosa/cosa_v1.go --package=cosa --root-type=Build" -- DO NOT EDIT
```

Comment:
I think we need to `grep -v` out this comment, not everyone's source code is going to be at `/src/cosa` - so I'll still end up with spurious diffs.

Not having file paths in generated code is part of [reproducible builds](https://wiki.debian.org/ReproducibleBuilds/).